Src/EB/AMReX_EB2_Level.H

```diff
@@ -496,6 +499,11 @@ GShopLevel<G>::define_fine (G const& gshop, const Geometry& geom,
                 }
             }
         }
+        total_multicuts = total_multicuts + nmulticuts;
+    }
+    if (plt_multiple_cuts){
```

Comment:
Move just inside the brace above (inside the `for iter` loop), such that this fails on the first iteration if there are multicuts. Also, like in the other aborts for multicuts, only do the abort if `cover_multiple_cuts` is false. After moving this, you won't need the `total_multicuts` variable, you can just use `nmulticuts`.